src/main/resources/beam-template.conf

```diff
@@ -691,7 +691,9 @@ beam.physsim.network.maxSpeedInference {
 beam.physsim.network.removeIslands = "boolean | true"
 
 # Replanning
+beam.replanning.replanningPenaltyInDollars = "double | 100.0"
```

Comment:
If we want to make beam more or less an internation product we can use something like `replanningMonetaryPenalty`. Though in this case we should change other references to dollar.